static/lang/re-en.json

```
@@ -1050,6 +1050,11 @@
             "CelestialResistance": {
                 "Prompt": "Select a type of energy associated with your bloodline."
             },
+            "Chronoskimmer": {
+                "DestabilizeFailureLabel": "Chronoskimmer Destabilize (Fail)",
+                "DestabilizeSuccessLabel": "Chronoskimmer Destabilize (Success)",
+                "StabilizeLabel": "Chronoskimmer Stabilize"
```

```suggestion
                "DestabilizeFailureLabel": "Destabilize timestream (Failure)",
                "DestabilizeSuccessLabel": "Destabilize timestream (Success)",
                "StabilizeLabel": "Stabilize timestream"
```
These sound better to me I think.